samples/bluetooth/peripheral_ams_client/README.rst

```diff
@@ -0,0 +1,276 @@
+   | Value        | 0,0.0,0.000 | Paused         |
+   +--------------+-------------+----------------+
+
+#. Set the :guilabel:`Apple Entity Update` value to ``02 03 00 31 32 30 2E 30 30 30``.
+#. Verify that the UART output is as follows::
+
+      AMS EU: 02,03,00 120.000
+
+   The following table explains the notification.
+
+   +--------------+---------+----------------+
+   | Field        | Value   | Interpretation |
+   +==============+=========+================+
+   | Entity ID    | 02      | Track          |
+   +--------------+---------+----------------+
+   | Attribute ID | 03      | Duration       |
+   +--------------+---------+----------------+
+   | Flags        | 00      |                |
+   +--------------+---------+----------------+
+   | Value        | 120.000 | 120 seconds    |
+   +--------------+---------+----------------+
+
+#. Set the :guilabel:`Apple Entity Update` value to ``02 02 00 6E 52 46 35 32 20 53 65 72 69 65 73 20 73 6F 6E 67``.
+#. Verify that the UART output is as follows::
+
+      AMS EU: 02,02,00 nRF52 Series song
+
+   The following table explains the notification.
+
+   +--------------+-------------------+--------------------+
+   | Field        | Value             | Interpretation     |
+   +==============+===================+====================+
+   | Entity ID    | 02                | Track              |
+   +--------------+-------------------+--------------------+
+   | Attribute ID | 02                | Title              |
+   +--------------+-------------------+--------------------+
+   | Flags        | 00                |                    |
+   +--------------+-------------------+--------------------+
+   | Value        | nRF52 Series song | Current song title |
+   +--------------+-------------------+--------------------+
+
+Playback
+^^^^^^^^
+
+#. Press **Button 2** to start audio playback.
+#. In nRF Connect for Desktop, verify that the :guilabel:`Apple Remote Command` is updated to ``02``.
+#. Set the :guilabel:`Apple Entity Update` value to ``00 01 00 31 2C 31 2E 30 2C 30 2E 30 31 34``.
+#. Verify that the UART output is as follows::
+
+      AMS EU: 00,01,00 1,1.0,0.014
+
+   The following table explains the notification.
+
+   +--------------+-------------+------------------------+
+   | Field        | Value       | Interpretation         |
+   +==============+=============+========================+
+   | Entity ID    | 00          | Player                 |
+   +--------------+-------------+------------------------+
+   | Attribute ID | 01          | Playback Info          |
+   +--------------+-------------+------------------------+
+   | Flags        | 00          |                        |
+   +--------------+-------------+------------------------+
+   | Value        | 1,1.0,0.014 | Playing at rate 1.0.   |
+   |              |             | Elapsed 0.014 seconds. |
+   +--------------+-------------+------------------------+
+
+Next Track
```

Comment:
```suggestion
Next track
```
Unless this is something that shows up in the UI. Otherwise, no title case, just sentence case.

Reply:
Will do.